### ISSUE TITLE
Build failure: rclip

### ISSUE THREAD

**Issue by willhansen** [NixOS/nixpkgs#287113]
### Steps To Reproduce

1. `nix-shell -p rclip`

### Build log

```
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/3rps3622myrjwaghjm987a1dsml57mmr-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tests/unit/test_model.py
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
[1m* Getting build dependencies for wheel...[0m
[1m* Building wheel...[0m
[1m[92mSuccessfully built [4mrclip-1.7.6-py3-none-any.whl[0m[1m[92m[0m
Finished creating a wheel...
Finished executing pypaBuildPhase
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for rclip-1.7.6-py3-none-any.whl
  - torch==2.0.1+cpu not satisfied by version 2.1.2
  - torchvision==0.15.2+cpu not satisfied by version 0.16.2

```

### Notify maintainers

@iynaix 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"

this path will be fetched (0.00 MiB download, 0.00 MiB unpacked):
  /nix/store/v8p3m2vgkcd3ykrdylm5vi9alrxbwhip-nix-info
copying path '/nix/store/v8p3m2vgkcd3ykrdylm5vi9alrxbwhip-nix-info' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.0-92-generic, Linux Mint, 21.3 (Virginia), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.19.3`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`

```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by iynaix** [comment]
This should be fixed in #281048, which was merged yesterday. :smile:

**Comment by iynaix** [comment]
rclip 1.7.24 is now available in nixos-unstable.
